WNPRC_Virology/resources/referenceStudy/folder.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<folder archiveVersion="22.0" label="Primate Electronic Health Record" xmlns="http://labkey.org/folder/xml">
```

Comment:
Should this be Viral Load module?

Reply:
yes, also there was an orphaned study.xml file from 21.11, removed it.